install/kubernetes/cilium/charts/hubble-relay/values.yaml

```diff
@@ -1,11 +1,23 @@
+enabled: false
+
+podSecurityPolicy:
+  enabled: false
+
+imagePullSecrets:
+
+serviceAccount:
+  create: true
+  annotations:
+
+debug: false
+
 # Configuration for Hubble Relay
 image:
   # repository of the docker image
-  repository: hubble-relay
+  repository: quay.io/isovalent/hubble-relay
   # tag is the container image tag to use
   tag: latest
-  # pullPolicy is the container image pull policy
-  pullPolicy: Always
+  pullPolicy: IfNotPresent
```

Comment:
Typically we have kept the development tree version of this as `pullPolicy: Always` and then whenever we branch for a release, we update the pull policy to `IfNotPresent`.